src/tests/omni_lock.rs

```diff
@@ -261,7 +547,109 @@ fn test_omnilock_transfer_from_ownerlock() {
     let mut senders = vec![sender0, sender1];
     for out_point in tx.input_pts_iter() {
         let sender = ctx.get_input(&out_point).unwrap().0.lock();
-        println!("code hash:{:?}", sender.code_hash());
+        // println!("code hash:{:?}", sender.code_hash());
+        assert!(senders.contains(&sender));
+        senders.retain(|x| x != &sender);
+    }
+    assert!(senders.is_empty());
+    assert_eq!(tx.outputs().len(), 1);
+    assert_eq!(tx.output(0).unwrap(), output);
+    assert_eq!(tx.output(0).unwrap().lock(), receiver);
+    let witnesses = tx
+        .witnesses()
+        .into_iter()
+        .map(|w| w.raw_data())
+        .collect::<Vec<_>>();
+    assert_eq!(witnesses.len(), 2);
+    assert_eq!(witnesses[0].len(), placeholder_witness0.as_slice().len());
+    assert_eq!(witnesses[1].len(), placeholder_witness1.as_slice().len());
+    ctx.verify(tx, FEE_RATE).unwrap();
+}
+
+#[test]
+fn test_omnilock_transfer_from_ownerlock_wl() {
+    let receiver = build_sighash_script(ACCOUNT2_ARG);
+    let sender1 = build_sighash_script(ACCOUNT1_ARG);
+    let hash = H160::from_slice(&sender1.calc_script_hash().as_slice()[0..20]).unwrap();
+    let mut cfg = OmniLockConfig::new_ownerlock(hash);
+
+    let mut ctx = init_context(
+        vec![(OMNILOCK_BIN, true)],
+        vec![(sender1.clone(), Some(61 * ONE_CKB))],
+    );
+    let (proof_vec, rc_root, rce_cells) = generate_rc(&mut ctx, cfg.id().to_smt_key().into());
```

Comment:
Since here we want test unlock from `admin mode`, so the `smt_key` must from `AdminConfig` but not from `OmniLockConfig`.

Reply:
And please check all other places have the same problem.